packages/eslint-plugin/src/rules/no-base-to-string.ts

```diff
@@ -42,16 +43,23 @@ export default util.createRule<Options, MessageIds>({
             type: 'boolean',
             default: true,
           },
+          ignoredTypeNames: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
+          },
         },
         additionalProperties: false,
       },
     ],
     type: 'suggestion',
   },
   defaultOptions: [{ ignoreTaggedTemplateExpressions: true }],
```

Comment:
I think it's 100% okay to default it to this, as that's what people would expect

```suggestion
  defaultOptions: [
    {
      ignoreTaggedTemplateExpressions: true,
      ignoredTypeNames: ['RegExp'],
    },
  ],
```

Reply:
done :)